storage/interface.go

```diff
@@ -190,6 +190,8 @@ type SelectHints struct {
 	Start int64 // Start time in milliseconds for this select.
 	End   int64 // End time in milliseconds for this select.
 
+	Limit int // Maximum number of results returned.
```

Comment:
Maybe we should explicitly mention that `0` means no limit (for minimal changes)